client/src/components/reviews/reviewListComponents/ReviewsList.jsx

```diff
@@ -3,14 +3,14 @@ import { useState, useEffect } from 'react';
 import {ReviewButtons} from './AdditionalButtons.jsx';
 import {ReviewTiles} from './ReviewTiles.jsx';
 import {SorterBar} from './SorterBar.jsx';
-import {ReviewsContainer} from '../sharedStyles/sharedStyledComponents';
+import {ReviewsContainer, ScrollableContainer} from '../sharedStyles/sharedStyledComponents';
 import data from '../apiExample.js';
 import axios from 'axios';
 import API_KEY from '../../../../../config.js'
 
 
 
-let ReviewsList = ( {productID} ) => {
+const ReviewsList = ( {productID} ) => {
   // dummyData
```

Comment:
This is for my benefit. What is the difference between let and const in this circumstance?

Reply:
'let' is more easily rewritten and changed. 'const' is supposed to be "constant" so it is harder to change. I was changing it to 'const' to be more consistent and because it does not need the ability to be overwritten/changed.

Reply:
Here is a really good article about var, let and const: https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/